### ISSUE TITLE
Not compatible with jdk 32 bit

### ISSUE THREAD

**Issue by nag123** [rockthejvm/udemy-spark-streaming#1]
Error running 'StreamingAggregations': Cannot run program "C:\Program Files\Java\jdk1.8.0_144\bin\java.exe" (in directory "D:\Learning\udemy-spark-streaming-master\udemy-spark-streaming-master"): CreateProcess error=216, This version of %1 is not compatible with the version of Windows you're running. Check your computer's system information to see whether you need a x86 (32-bit) or x64 (64-bit) version of the program, and then contact the software publisher

**Comment by daniel-ciocirlan** [comment]
This is probably due to the JDK installation. Do you have Windows 32bit?

**Comment by nag123** [comment]
Yes , Its a 32 bit Windows Operating System. And I installed x86  version of java to my windows.

**Comment by nag123** [comment]
> Yes , Its a 32 bit Windows Operating System. And I installed x86 version of java to my windows.

**Comment by nag123** [comment]
> This is probably due to the JDK installation. Do you have Windows 32bit?

Yes , Its a 32 bit Windows Operating System. And I installed x86 version of java to my windows.
